gpii/node_modules/preferencesServer/src/preferencesServer.js

```diff
@@ -52,6 +52,52 @@ fluid.defaults("gpii.preferencesServer", {
             route: "/preferences/:gpiiKey",
             method: "put",
             type: "gpii.preferencesServer.put.handler"
+        },
+        // GPII-2966
+        prefsSafeGet: {
```

Comment:
Update documentation with these new endpoints.

Reply:
All the endpoints should be in the doco now.